pkg/csi/cinder/openstack/openstack.go

```diff
@@ -84,6 +100,18 @@ func (cfg Config) toAuthOptions() gophercloud.AuthOptions {
 	}
 }
 
+func (cfg Config) toAuth3Options() tokens3.AuthOptions {
```

Comment:
only v3 is supported in csi, so no need to create new cfg here , we can reuse existing one

Reply:
are you meaning cinder v3 ?

Reply:
@ramineni hi, any related doc mentions this(only v3 is supported in csi)?

Reply:
@dricoco _gophercloud.AuthOptions_ is intended to support for v2 and v3 both https://github.com/gophercloud/gophercloud/blob/master/auth_options.go#L30
IMO no need to add this  _tokens3.AuthOptions_ seperately.